src/transaction/locator_cl.c

```diff
@@ -5904,6 +5931,12 @@ locator_remove_class (MOP class_mop)
 	{
 	  goto error;
 	}
+
+      error_code = remove_lob_dir (insts_hfid, -1);
```

Comment:
가독성: -1 값을 의미를 가진 매크로로 정의해서 사용 필요
```suggestion
      error_code = remove_lob_dir (insts_hfid, -1);
```